src/Iesi.Collections/Iesi.Collections.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard1.3;net40;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard1.3;netstandard2.0;net40;net461</TargetFrameworks>
```

Comment:
`netstandard2.0` target added because `netstandard1.3` requires additional reference to `System.Runtime.Serialization.Formatters` which I want to avoid as it is ".NET Standard + Platform Extensions"